hw11/utils/webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const args = process.argv;
+const isFileCSS = args.includes('--styles');
+// const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
why need it here?

Reply:
because it is from task:
'3. Настроить ExtractTextPlugin таким образом, чтобы при генерации файла css....'
and i tried to do this... but it isn't supported for webpack higher 4.0.0